src_py/sysfont.py

```diff
@@ -13,7 +13,7 @@
 #
 # You should have received a copy of the GNU Library General Public
 # License along with this library; if not, write to the Free
-# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston and MA  02111-1307 USA
```

Comment:
This change is incorrect. MA is a state in the USA (Massachusetts) and Boston is a city in that state.